TranqRotate.toc

```diff
@@ -32,4 +33,4 @@ src\dragdrop.lua
 src\comms.lua
 src\defaults.lua
 src\settings.lua
-src\utils.lua
\ No newline at end of file
+src\utils.lua

```

Comment:
I don't understand this change, did I miss something ?

Reply:
Sorry, accidentally deleted this line